recipes/pthread-stubs/meta.yaml

```diff
@@ -0,0 +1,29 @@
+{% set version = "0.1" %}
+{% set md5 = "774eabaf33440d534efe108ef9130a7d" %}
+
+package:
+  name: pthread-stubs
+  version: {{ version }}
+
+source:
+  fn: libpthread-stubs-{{ version }}.tar.bz2
+  url: http://xcb.freedesktop.org/dist/libpthread-stubs-{{ version }}.tar.bz2
+  md5: {{ md5 }}
+
+build:
+  skip: true  # [win or osx]
+  number: 0
+
+test:
+  commands:
+    - test -f $PREFIX/lib/pkgconfig/pthread-stubs.pc
+
+about:
+    home: http://xcb.freedesktop.org/
+    license: MIT
```

Comment:
Can we please include the license file here so as to comply with the MIT license?